build.gradle

```diff
@@ -40,6 +40,8 @@ dependencies {
     compile("org.springframework.security:spring-security-oauth2-resource-server")
     compile("org.springframework.security:spring-security-oauth2-jose")
```

Comment:
You can remove the rest of these dependencies if you add our starter. It pulls them in.